src/formatDate.js

```diff
@@ -8,7 +8,89 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let result = '';
+
+  let symbol = '';
+
+  for (let i = 0; i < date.length; i++) {
+    if (isNaN(date[i])) {
+      symbol = date[i];
+    }
+  }
+
+  const slicedDate = date.split(symbol);
+
+  const indexOfDay = fromFormat.indexOf('DD');
+  const indexOfMonth = fromFormat.indexOf('MM');
+  const indexOfYear = findindexOfYear(fromFormat, slicedDate);
```

Comment:
The function `findindexOfYear` is not defined before its usage. Make sure to define the function before calling it to avoid reference errors.